netplan/cli/commands/apply.py

```diff
@@ -251,17 +251,16 @@ def command_apply(self, run_generate=True, sync=False, exit_on_error=True):  # p
             utils.systemctl_network_manager('start', sync=sync)
 
     @staticmethod
-    def is_composite_member(composites, phy):  # pragma: nocover (covered in autopkgtest)
+    def is_composite_member(composites, phy):
         """
         Is this physical interface a member of a 'composite' virtual
         interface? (bond, bridge)
         """
         for composite in composites:
-            for _, settings in composite.items():
-                members = settings.get('interfaces', [])
```

Comment:
I think we should rather check something like this before `members = settings.get('interfaces', [])`:
```
if not type(settings) is dict:
    continue
```

Reply:
Yes that is a much more sensible fix, I had mistaken what the arguments were. Thanks

Reply:
Could you update that accordingly? As I do not want to do a self-review & merge of this change.

Reply:
I've pushed that change now.